### ISSUE TITLE
Add to autorun

### ISSUE THREAD

**Issue by janmechtel** [janmechtel/rockettype#7]
- Add to auto-run on Windows start

Easiest might be to do this through the installer for now:

`CreateShortCut “$SMSTARTUP\Monitor.lnk” “$INSTDIR\Monitor.exe”`

https://stackoverflow.com/questions/48317822/let-installer-create-an-auto-run-shortcut-on-windows-startup-for-only-one-user#48317933

**Comment by appins** [comment]
Oh, I should've looked at this issue because this seems rather simple and I was kind of dreading it. 

I was thinking perhaps it might be a good idea to have a checkbox in the app that can turn the autorun on/off without having to use the task manager. For now, I'll see if I can add it to the NSIS installer.

**Comment by appins** [comment]
Yeah, I'm having trouble with adding that. It is seemingly doing nothing. Perhaps adding a checkbox though the python app would be the best way to go.

What do you think? 

Also, my apologies for the delay with this issue. Please see the comments on Upwork.

**Comment by janmechtel** [comment]
What did you try so far with the NSIS? Why is it not working? Maybe I can give it a try

**Comment by appins** [comment]
This is what I got so far:
![image](https://user-images.githubusercontent.com/13598541/93267905-3584b780-f761-11ea-953f-af2d6cb04008.png)

There was no checkbox and when I had the code outside of the function that didn't create the shortcut either. I feel like I'm missing something...
